controllers/cloud.redhat.com/config/types.go

```diff
@@ -509,6 +509,8 @@ type DeploymentMetadata struct {
 
 	// Name of deployment
 	Name string `json:"name"`
+
```

Comment:
Does this need to be here? For a start, this is only necessary for cdappconfig.json output, secondly if you want to edit this file, you should edit the json schema and run the converter using `make genconfig`

Reply:
Must have been a mistake and I was trying to edit clowdapp_types.go and edited the wrong file. Reverted in next commit.